### PR TITLE
Update dictionaries

### PR DESCRIPTION
**Pull request by NilsEnevoldsen**
Notable changes listed in each commit.

### CODE REVIEW

File: dictionary/get_vdem.R [hunk 1]
```diff
@@ -1,12 +1,17 @@
 source(here::here('dictionary/utilities.R'))
 
-vdem <- readRDS('dictionary/data_raw/data_vdem_v8_april2018.rds') %>%
+# readRDS("~/countrycode/dictionary/data_raw/V-Dem-CY-Core-v12.rds") %>%
+#   arrange(country_text_id,year) %>%
+#   select(country_name,country_id,year) %>%
+#   saveRDS(vdem, file = "~/countrycode/dictionary/data_raw/data_vdem_v12_march2022.rds", version = 2, compress = "xz")
```
Comment (cjyetman) [inline dictionary/get_vdem.R:6]: maybe get rid of this commented code so it doesn't confuse us later?

Comment (NilsEnevoldsen) [reply dictionary/get_vdem.R:6]: I included it deliberately. `data_vdem_v8_april2018.rds` appears to have been a reduced version of the V-Dem dataset. This code reminds us how to munge the V-Dem dataset into the reduced version. For example, it took me a little while to understand that the dataset was sorted on a variable that was no longer present in the reduced dataset.

**Comment by vincentarelbundock** [comment]
FYI there's a merge conflict with your previous PRs, but I'm not sure which should have precedence in the merge conflict.

**Comment by NilsEnevoldsen** [comment]
The RDAs will need to be regenerated regardless. Let this pull request marinate for a few days, I think. I want to fix Trinidad y Tobago anyway. I also just noticed that the UN website doesn't follow [GEGN.2/2021/CRP.130/Rev.1](https://unstats.un.org/unsd/ungegn/sessions/2nd_session_2021/documents/GEGN.2_2021_CRP130_4c_WG_Country_Names_rev2.pdf) for Guinea-Bissau. (No hyphen at https://www.un.org/en/about-us/member-states#gotoG.) I'm tempted to do a deeper exploration of GEGN.2/2021/CRP.130/Rev.1.

**Pull request closed without merging**